integration-tests/test-class-admin-asset.php

```diff
@@ -57,7 +57,7 @@ public function test_constructor_default_values() {
 		$this->assertEquals( 'name', $asset->get_name() );
 		$this->assertEquals( 'src', $asset->get_src() );
 		$this->assertEquals( [], $asset->get_deps() );
-		$this->assertEquals( WPSEO_VERSION, $asset->get_version() );
+		$this->assertEquals( '', $asset->get_version() );
```

Comment:
The test states that it also covers `get_suffix`, but it doesn't 😕

Reply:
Fixed, removed that `@covers`, it's covered below.